spec/models/export_spec.rb

```diff
@@ -38,6 +38,18 @@
       expect(results.first).to eq 'Account address,Show boosts,Notify on new posts,Languages'
       expect(results.second).to eq 'one@local.host,true,false,'
     end
+
+    it 'returns a csv of the following tags' do
+      tag_names = 2.times.map { |_| Fabricate(:tag_follow, account: account).tag.name }
+
+      export = Export.new(account).to_following_tags_csv
+      results = export.strip.split("\n")
+
+      expect(results.size).to eq 3
```

Comment:
Line 48 - 51 can be substitued with:

```suggestion
      expect(results).to match_array(['Tag', tag_names.first, tag_names.second])
```